apps/web/pages/[user]/[type].tsx

```diff
@@ -165,6 +165,7 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
   const eventData = await ssr.viewer.public.event.fetch({
     username,
     eventSlug: slug,
+    isTeamEvent: false,
```

Comment:
`isTeamEvent` was missing and this was the reason this prefetched query response was unused during hydration